tests/data/validation_problems.po

```diff
@@ -49,3 +49,17 @@ msgstr[1] "2. Estas {num} objectos"
 # msgid_plural "3. There are {num} things"
 # msgstr[0] "3. Estas num objectos"
 # msgstr[1] "3. Estas {num} objectos"
+
+# These should not raise a validation error, because LONG_DATE_FORMAT and
+# friends are special.
+msgid "LONG_DATE_FORMAT"
```

Comment:
For example, there was a time there was a string in edx-platform that was
causing issues because it was an instructor facing string explaining how to
use json properly. So, it would get erroneously validated all the time. I
would have loved a generic way to quickly whitelist that specific string.

On Thu, Dec 17, 2015 at 11:38 AM, Sarina Canelake <sarina@edx.org> wrote:

> I guess what I'm asking is, this PR is whitelisting specific strings for
> edx-platform, which seems wrong. Wouldn't it be better to allow a variable
> whitelist somewhere in your configuration, so that you can quickly extend
> the whitelist for specific things in whatever repo the tool is running on?
> As we move to IDAs this toolset is being used more generically across more
> repos.
>
> On Thu, Dec 17, 2015 at 11:33 AM, Ned Batchelder <notifications@github.com
> > wrote:
>
>> In tests/data/validation_problems.po
>> <https://github.com/edx/i18n-tools/pull/35#discussion_r47927091>:
>>
>> > @@ -49,3 +49,17 @@ msgstr[1] "2. Estas {num} objectos"
>> >  # msgid_plural "3. There are {num} things"
>> >  # msgstr[0] "3. Estas num objectos"
>> >  # msgstr[1] "3. Estas {num} objectos"
>> > +
>> > +# These should not raise a validation error, because LONG_DATE_FORMAT and
>> > +# friends are special.
>> > +msgid "LONG_DATE_FORMAT"
>>
>> This is a low-tech way to identify these names. I guess I could just make
>> a list instead?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/edx/i18n-tools/pull/35/files#r47927091>.
>>
>
>